packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -659,6 +663,10 @@ export const Table = <
     setPageSize(pageSize);
   }, [pageSize, setPageSize]);
 
+  const numSelectedRows =
```

Comment:
Wrap this with `React.useMemo` or move under `paginatorRendererProps` because right not it is going trough all rows on each rerender.

Reply:
I don't maybe it's just for me but `selectedRowsCount` sounds better.

Reply:
On that note, can we maybe make it to `totalSelectedRowsCount` to be the same name as the property on [line 97](https://github.com/iTwin/iTwinUI-react/pull/837/files#diff-9edd44b431060c27e0260d286ee188bdf901225b1ee84ff6540f9d796dd0f999R97)? Or maybe make changed line 97 to `selectedRowsCount` so both have the same names?

Reply:
I have moved this count under `paginatorRendererProps` and removed the `numSelectedRows` variable; however, now the row selection count is not updating in the unit test which is causing it to fail. I'm still looking into this